EarlGrey/Action/GREYActions.m

```diff
@@ -261,6 +254,68 @@ @implementation GREYActions
 
 #pragma mark - Private
 
+
+/**
+ * Get WebView input text value.
+ * @param element The element to target
+ */
++ (NSString*)grey_webviewInputValue: (id) element {
```

Comment:
nit:
no space after : or (id)

grey_webviewInputValue:(id)element

fyi, the code formatting here follows the google objc styleguide: https://google.github.io/styleguide/objcguide.xml